Jenkinsfile

```diff
@@ -107,6 +122,30 @@ def cancel_previous_build() {
 
 cancel_previous_build()
 
+stage('Prepare') {
+  node('master') {
+    // When something is provided in ci_*_param, use it, otherwise default with ci_*
```

Comment:
just curious why not to use defaultValue instead of doing this? we could rename e.g. ci_cpu to prod_ci_cpu or something.

Reply:
That was my original plan, but Jenkins has these quirks... we can set the default parameter, and that will be parsed/adopted by the UI once. To update that later, it needs to be done via the UI, hence we can't count on these defaults to be set _as code_.

Reply:
aha, i see. seems reasonable to me.